### DISCUSSION TITLE
Title for the music piece

### DISCUSSION THREAD

**Discussion by anjanai** [Studio-kalavati/Swaranotebook-bandish-editor#112] [Ideas / Feature Requests]
Would be nice to have a title for each "bandish".

**Comment by shark8me** [comment]
The first time when you save a Bandish , there's a popup whether you can
enter a title.
It shows up both on the URL, and the "my notations" menu (by clicking on
the hamburger button).

Did you instead prefer to see the title of the bandish just above where the
Svaras are typed?

**Comment by anjanai** [comment]
Yes, thanks, I meant I'd like to see the title on the bandish page, so when
I take a screenshot or a printout, the title can be seen.

**Comment by shark8me** [comment]
Implementation note: 
This can be done by implementing #107 , the first label can be title or the section header (e.g. Sthayi or Antara)

**Comment by shark8me** [comment]
<img width="1113" height="819" alt="image" src="https://github.com/user-attachments/assets/67920a18-75e6-4095-9eb4-5dfce24ba47b" />

This feature is available since Aug 18 2025. 

The title of the Bandish is shown at the top of the page, and can be edited by clicking on the :pen: icon
